python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -253,6 +229,18 @@ class RandomForestRegressor(BaseRandomForestModel,
         module level, `cuml.global_settings.output_type`.
         See :ref:`output-data-type-configuration` for more info.
 
+    Notes
+    -----
+    **Known Limitations**\n
+    This is an early release of the cuML
+    Random Forest code. It contains a few known limitations:
+
+      * GPU-based inference is only supported if the model was trained
+        with 32-bit (float32) datatypes. CPU-based inference may be used
+        in this case as a slower fallback.
```

Comment:
I would suggest adding that the alternatives are either using CPU inference for float64 or letting automatic dtype conversion happen during inference (which it is set to do by default)